voice/bxml/callbacks/conferenceMemberExit.md

```diff
@@ -14,6 +14,8 @@ HTTP/1.1 204
 | conferenceId | The id for the conference that was joined                                                                       |
 | name         | The custom name used to reference this conference.                                                              |
 | callId       | The callId of the member that left the conference.                                                              |
+| from         | The from number from the original call                                                                          |
+| to           | The to number from the original call                                                                            |
```

Comment:
Please also update the example below.

We could also use a more realistic conferenceId, like:
conf-59082d52-4a2ab5be-ce26-43ed-af94-431b8a19d4e3